crates/contract/src/lib.rs

```diff
@@ -2836,4 +2842,98 @@ mod tests {
         let result = contract.migration_info();
         assert_eq!(result, expected_vals);
     }
+
+    fn propose_and_vote(
+        contract: &mut MpcContract,
+        update: Update,
+        expected_update_id: u64,
+    ) -> Vec<dtos::AccountId> {
+        let update_id = contract.proposed_updates.propose(update.clone());
+        assert_eq!(update_id.0, expected_update_id);
+        // generate two accounts for voting
+        let account_id_0 = gen_account_id();
+        let account_id_1 = gen_account_id();
```

Comment:
Given the [discussion](https://nearone.slack.com/archives/C07UW93JVQ8/p1763644116745819?thread_ts=1763628576.510629&cid=C07UW93JVQ8) brought up by @gilcu3, it might be worth making the tests easily parameterized over the number of accounts, if possible. It would also reduce the copy-paste. Not pushing for it though, since we already have plenty of tests with a fixed number of participants/accounts/whatever.